USAGE.md

```diff
@@ -115,33 +115,51 @@ If `citation-tags.tsv` defines the tag `study-x`, then this study can be cited l
+[`content/manual-references.json`](content/manual-references.json) is the default file to specify custom CSL JSON metadata.
+Manual references are matched to citations using their "id" field.
+For example, to manually specify the metadata for the citation `@url:https://github.com/manubot/rootstock`, add a CSL JSON Item to `manual-references.json` that contains:
 
 ```json
-"standard_citation": "url:https://github.com/greenelab/manubot-rootstock"
+"id": "url:https://github.com/manubot/rootstock",
 ```
 
-The metadata for `raw` citations must be provided in `manual-references.json` or an error will occur.
-For example, to cite `@raw:private-message` in a manuscript, a corresponding CSL Item in `manual-references.json` is required, such as:
+The metadata for `raw` citations must be provided in a manual reference file (e.g. `manual-references.json`) or an error will occur.
+For example, to cite `@raw:private-message` in a manuscript, a corresponding CSL Item is required, such as:
 
 ```json
 {
+  "id": "raw:private-message",
   "type": "personal_communication",
-  "standard_citation": "raw:private-message",
   "title": "Personal communication with Doctor X"
 }
 ```
 
-All references provided in `manual-references.json` must provide values for the `type` and `standard_citation` fields.
+All references provided in `manual-references.json` must provide values for the `id` and `type` fields.
 For guidance on what CSL JSON should be like for different document types, refer to [these examples](https://github.com/aurimasv/zotero-import-export-formats/blob/a51c342e66bebd97b73a7230047b801c8f7bb690/CSL%20JSON.json).
 
+Manubot offers some support for other bibliographic metadata formats besides CSL JSON, by delegating conversion to the `pandoc-citeproc --bib2json` [utility](https://github.com/jgm/pandoc-citeproc/blob/master/man/pandoc-citeproc.1.md#convert-mode).
+Formats are inferred from filename extensions.
+So, for example, to provide metadata for `@url:https://github.com/manubot/rootstock` in BibTeX format, create the file `content/manual-references.bib` and create an item whose definition starts with:
+
+```latex
+@misc{url:https://github.com/manubot/rootstock,
+```
+
+Processed reference metadata in CSL JSON format, either generated by Manubot or specified via manual references, is exported to `references.json`.
+This file is located in the `output` branch on GitHub or in the `output` subdirectory of local builds.
+The "id" field in `references.json` and in the final manuscript uses a shortened ID that is derived from the original ID.
+For debugging information, see `citations.tsv`, which shows citation identifiers as they progress through the processing pipeline.
+In order to freeze all references, rather than have Manubot regenerate them during future builds, copy the output `references.json` file to `content` as a manual references file.
```

Comment:
You could remind users of the filename conventions here by ending the sentence with "that matches the `manual-references*.*` pattern."